test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -1995,9 +2000,9 @@ TEST_F(SslContextStatsTest, IncOnlyKnownCounters) {
   // fallback registration does not occur. So we test for the fallback only in
   // release builds.
 #ifdef NDEBUG
-  cipher = store_.findCounterByString("ssl.ciphers.fallback");
-  ASSERT_TRUE(cipher.has_value());
-  EXPECT_EQ(1, cipher->get().value());
+  stat = store_.findCounterByString("ssl.ciphers.fallback");
```

Comment:
`stat` is not defined in this scope, you need to add `Stats::CounterOptConstRef ...`.

Reply:
yeah when hand-testing this you want to run this unit test with "-c opt"

Reply:
Yeah :/ I had only tested in debug and fastbuild.